libs/pmp-web/core/src/lib/modules/pmp-store.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { EffectsModule } from '@ngrx/effects';
+import { NxModule } from '@nrwl/angular';
+import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+import { StoreModule } from '@ngrx/store';
+
+@NgModule({
+  imports: [
+    EffectsModule.forRoot([]),
+    NxModule.forRoot(),
+    StoreModule.forRoot(
+      {},
+      {
+        runtimeChecks: {
+          strictActionImmutability: true,
+          strictStateImmutability: true
+        }
+      }
+    ),
+    StoreDevtoolsModule.instrument({
+      maxAge: 25
```

Comment:
What about production mode?

Reply:
Would it be enough to set logMode to true for every environment?